### PR TITLE
Add Excise to the types which may express a unit

### PR DESCRIPTION
**Pull request by jebw**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-825

### What?

This PR extends the list of unit expressable measures to include measures of type series Q - ie Excise

- [x] Added `Q` to the list of unit expressable measures
- [x] Changed the specs to explicitly test for each of the unit expressable measures
- [x] Feature flagged this change

### Why?

I am doing this because:

- The duty calculator will require a list of applicable measure units for Excise duties

### CODE REVIEW

**Comment by jebw** [comment]
```json
"meta": {
      "duty_calculator": {
        "applicable_additional_codes": {},
        "applicable_measure_units": {
          "LTR": {
            "measurement_unit_code": "LTR",
            "measurement_unit_qualifier_code": "",
            "abbreviation": "l",
            "unit_question": "What is the volume of the goods you will be importing?",
            "unit_hint": "Enter the value in litres",
            "unit": "litres",
            "measure_sids": [
              -485461,
              -485453,
              -485455,
              -485456,
              -485457,
              -485458,
              -485459,
              -485460
            ]
          }
        },
        "applicable_vat_options": {
          "VAT": "Value added tax (20.0%)"
        },
        "entry_price_system": false,
        "meursing_code": false,
        "source": "uk",
        "trade_defence": false,
        "zero_mfn_duty": false
      }
```

**Review by willfish — APPROVED** [review]

**Comment by jebw** [comment]
@willfish Cheers for the review, still WIP at the moment - blows up when using the commodity codes Matt had listed so I need to look into that before merging anything

File: app/models/measure_type.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ class MeasureType < Sequel::Model
     'C', # Applicable duty
     'D', # Anti-dumping/countervailing measures
     'J', # Countervailing charge
+    'Q', # Excise
```
Comment (willfish) [inline app/models/measure_type.rb:31]: We need to add conditionality around the feature flag for this

File: app/models/measure_type.rb [hunk 2]
```diff
@@ -24,11 +24,20 @@ class MeasureType < Sequel::Model
...
+      'D', # Anti-dumping/countervailing measures
+      'J', # Countervailing charge
+      'Q', # Excise
+    ].freeze
+  else
+    UNIT_EXPRESSABLE_MEASURES = [
+      'C', # Applicable duty
+      'D', # Anti-dumping/countervailing measures
+      'J', # Countervailing charge
+    ].freeze
```
Comment (tavy88) [inline app/models/measure_type.rb:39]: Not too precious about it but how would you feel adding 
```ruby
      'C', # Applicable duty
      'D', # Anti-dumping/countervailing measures
      'J', # Countervailing charge
```
to a constant and then you do something like:
```ruby
if Rails.application.config.include_excise_measure_units
    UNIT_EXPRESSABLE_MEASURES =  UNIT_EXPRESSABLE_MEASURES + [
      'Q', # Excise
    ].freeze
  else
    UNIT_EXPRESSABLE_MEASURES 
```
?

**Review by tavy88 — APPROVED** [review]

File: config/application.rb [hunk 3]
```diff
@@ -42,5 +42,8 @@ class Application < Rails::Application
       Sequel::Model.db.extension :server_block
       Sequel::Model.db.extension :auto_literal_strings
     end
+
+    # Feature flags
+    config.include_excise_measure_units = ENV.fetch('INCLUDE_EXCISE_MEASURE_UNITS', 'false') == 'true'
```
Comment (willfish) [inline config/application.rb:47]: So the idiom in this app is to put these kinds of feature flags in the TradeTariffBackend module. I reckon just for consistency we should move this there?

**Review by willfish — APPROVED** [review]

**Pull request merged**